modules/features/manatime_feature_config/manatime_feature_config.module

```diff
@@ -0,0 +1,17 @@
+<?php
+/**
+ * @file
+ * Drupal needs this blank file.
+ */
+
+ /**
+  * Function to add aliases path.
+  */
+function add_path_alias($source, $alias, $language) {
```

Comment:
All functions should be prefixed with module name; so, rename to `manatime_feature_config_add_path_alias`